ROADMAP.md

```diff
@@ -0,0 +1,103 @@
+# OCI Specs Roadmap
+
+This document serves to provide both a long term roadmap on our quest to a 1.0 version of the OCI container specification.
+Its goal is to help both maintainers and contributors find meaningful tasks to focus on and create a low noise environment.
+The items in the 1.0 roadmap can be broken down into smaller milestones that are easy to accomplish.
+The topics below are broad and small working groups will be needed for each to define scope and requirements or if the feature is required at all for the OCI level.
+Topics listed in the roadmap do not mean that they will be implemented or added but are areas that need discussion to see if they fit in to the goals of the OCI.
+
+## 1.0
+
+### Digest and Hashing
+
+A bundle is designed to be moved between hosts. 
+Although OCI doesn't define a transport method we should have a cryptographic digest of the on-disk bundle that can be used to verify that a bundle is not corrupted and in an expected configuration.
```

Comment:
Since I think there may be a difference between the "on-disk" bundle and the "archive" that we create when we try to move a bundle I think it might be best to remove the "on-disk" part of this sentence.  For example, an incoming bundle might have the notion of layers, but if the implementation of OCI flattens its down to just a single dir I don't think we want to hash the flatten dir, we want to hash the original bundle and do verification of that.